### DISCUSSION TITLE
Causal inference + X

### DISCUSSION THREAD

**Discussion by fenzhantw** [Pseudo-Lab/9th-builder#18] [plan]
## 프로젝트 소개
인과 추론은 다양한 분야에서 널리 활용되고 있습니다. 간단하게 예를 들어볼까요?

1. 추천 시스템에서는 사용자와 가장 상호작용할 가능성이 높은 상품을 추천하지만, 인기도 편향과 위치 편향 같은 다양한 편향이 학습을 방해합니다. 인과 추론의 개념을 활용해 이러한 편향의 원인을 파악하고, 편향을 교정(debiasing)할 수 있습니다. 예를 들어, 인기도 편향을 활용한 추천 시스템에서의 Causal Intervention 에 관한 연구가 진행된 바 있습니다. 
- [[Causal Intervention for Leveraging Popularity Bias in Recommendation](https://arxiv.org/abs/2105.06067)]
- [[Causal Inference in Recommender Systems: A Survey and Future Directions](https://arxiv.org/abs/2208.12397)]

2. 공정성 평가에도 인과 추론이 활용됩니다. 머신러닝 모델의 공정성을 평가할 때, 인과 추론을 통해 불공정한 요인을 분석하고 개선할 수 있죠. 
- [[The Causal Fairness Field Guide](https://www.frontiersin.org/journals/big-data/articles/10.3389/fdata.2022.892837/full)]

3. LLM은 인과 추론에서 큰 역할을 하기도 합니다. 예를 들어, 관찰 데이터에서 변수 간의 인과 효과를 추정하는 Causal Discovery에서 대형 언어 모델(LLM)의 능력을 활용할 수 있습니다. 
- [[From Query Tools to Causal Architects: Harnessing Large Language Models for Advanced Causal Discovery from Data](https://arxiv.org/abs/2306.16902)]
- [[Efficient Causal Graph Discovery Using Large Language Models](https://arxiv.org/abs/2402.01207)]

4. 특정 도메인에서도 인과 추론은 활발히 사용됩니다. 예를 들어, 제조 운영 관리에서는 공정상의 문제를 해결하기 위한 근본 원인 분석(Root Causal Analysis)에 인과 추론이 많이 활용됩니다. 또한, 대량의 시계열 데이터를 분석해 인과 관계를 파악하는 연구도 각광받고 있습니다.
- [[Root cause diagnosis for process faults based on multisensor time-series causality discovery](https://www.sciencedirect.com/science/article/abs/pii/S0959152422002293)]

이처럼 인과 추론은 다양한 분야에서 상호작용할 수 있습니다. 따라서 이번 스터디의 목표는 스터디원이 각자 원하는 'X'를 찾아 이에 대한 목표를 설정하고, 개인 프로젝트를 수행하는 것입니다.

위에서 제시한 주제뿐만 아니라, 인과 추론을 활용한 챗봇을 개발해 보거나, 아키텍처에 관심이 있다면 실험 플랫폼을 구축해 보는 것도 좋은 선택이 될 것입니다.

**Causal inference에 대한 대주제만 같으면, 모든 세부 주제로 X를 선정할 수 있습니다.**

### 프로젝트 목표
1. Causal inference + X의 "X" 공유
2. X에 대한 목표 및 프로젝트 수행
3. X에 대한 프로젝트 성과 공유

### 프로젝트 내용
- 본인의 관심사에 맞는 X를 발견하기 (인과추론 응용, 시계열, Causal Discovery, NLP, LLM, RAG Evaluation)
- 학습한 인과추론 이론이 어떻게 X와 상호작용할 수 있는지에 대한 이해
- X를 활용하여 자신만의 소소한 프로젝트 결과물 만들기

## 모집 대상
- 인과추론에 대해 이해도가 있으신 분

**지원 대상**
- 기초 인과추론 개념에 대한 이해가 있으신 분
- 현업에서 인과추론 수행해보신 분 또는 적용할 계획이 있으신 분
- 영어로 된 자료에 익숙하신 분
- 시간을 투자하여 자습을 할 수 있으신 분
- **✨스터디에서 공부하는 내용을 잘 정리해나갈 의지가 있으시고 스터디에 참여 할 여유시간이 되시는 분✨**

**모집 인원**
- 0명
- 기존 인원이 10명(빌더 포함)이 있기 때문에 이번 차수에는 모집하지 않습니다.

### 첫 모임 날짜 & 시간 & 장소

- 날짜: 2024.09.03 (화요일) 
- 시간: 19:00 - 22:00
- 장소: 오프라인 - 프로젝트 시작 동기 및 목표 소개

### 정기 모임 시간 & 장소

- 시간: 매주 화요일 저녁 21시 00분 (예상 소요 시간: 60-90분)
- 장소: 온라인
    - 협의하에 한 달에 1회 정도의 오프라인 모임

### 학습 자료

- **인과 추론 실무에 관한 모든 자료(Paper, 블로그 글 등 등)**
    1. https://research.netflix.com/research-area/experimentation-and-causal-inference
    2. [[KDD 2023 Workshop - Causal Inference and Machine Learning in Practice](https://causal-machine-learning.github.io/kdd2023-workshop/)](https://causal-machine-learning.github.io/kdd2023-workshop/)
    3. https://www.nabe.com/tec2022
    4. https://www.uber.com/en-KR/blog/causal-inference-at-uber/
    6. [[**PDAMS 23](https://www.youtube.com/watch?v=rkP4TK1SPVQ)](https://www.youtube.com/watch?v=rkP4TK1SPVQ)**

### 스터디 활동 방법
💡 **오픈 아카데미** 형태(=청강 가능)로 진행하며, 발표자료는 가짜연구소를 통해 **공개적으로 배포할** 수 있도록 할 예정입니다.
    
1. **진행 & 발표**
    - 출석체크를 위해 실명으로 참여 부탁드립니다.
    - 온라인에서 여러분을 만나는 환경이므로 웹캠을 꼭 준비해 주세요~
    - 발표자는 OT 날 선정 (참여 인원에 따라 1인 1-2회 정도 발표 예상)
    
2. **학습 활동 (필수)**
    - **학습 내용에 대한 질문 및 공유 여러분들의 목소리가 너무 듣고 싶어요**

## 주차별 목표와 학습 내용
💡 주차별 진행계획은 러너(스터디원)분들과 협의에 의해 변경될 수 있습니다.

| 날짜 | 내용 | 발표자 | 매체 | 
| ----- | ----- | -------- | ----- |
| 2024/09/03| OT (스터디 소개, 방향 논의),  | [김성수]() | [링크]() | 
| 2024/09/10| OT (스터디 소개, 방향 논의), Causal Pattern 찾기|  [김성수]() | [링크]() | 
| 2024/09/24| Causal Inference + X |  []() | [링크]() |
| 2024/10/01 | Causal Inference + X |  []() | [링크]() |
| 2024/10/08 | Causal Inference + X |  []() | [링크]() |

**Comment by jsshin2022** [comment]
와.... 이번에도 너무나 기대됩니다 성수님 👍 하반기도 같이 화이팅하시죠!
